crates/sui-bridge/abi/sui_bridge.json

```diff
@@ -355,11 +421,6 @@
         "internalType": "address",
         "name": "_limiter",
         "type": "address"
-      },
-      {
-        "internalType": "address",
-        "name": "_wETH",
-        "type": "address"
```

Comment:
this @Bridgerz

Reply:
`SuiBridge` no longer has a reference to weth as the vault handles all wrapping and unwrapping. must be a stale reference to an old version.